src/main/java/org/elasticsearch/index/query/CommonTermsQueryBuilder.java

```diff
@@ -133,17 +155,25 @@ public CommonTermsQueryBuilder boost(float boost) {
         return this;
     }
 
+    public float boost() {
+        return boost;
+    }
+
     /**
      * Sets the cutoff document frequency for high / low frequent terms. A value
      * in [0..1] (or absolute number >=1) representing the maximum threshold of
      * a terms document frequency to be considered a low frequency term.
      * Defaults to
-     * <tt>{@value CommonTermsQueryParser#DEFAULT_MAX_TERM_DOC_FREQ}</tt>
+     * <tt>{@value #DEFAULT_CUTOFF_FREQ}</tt>
      */
-    public CommonTermsQueryBuilder cutoffFrequency(float cutoffFrequency) {
-        this.cutoffFrequency = cutoffFrequency;
+    public CommonTermsQueryBuilder cutoffFrequency(Float cutoffFrequency) {
```

Comment:
I think float was better here, maybe we can do the other way around and leave the setter as-is and change the type of the `cutoffFrequency` to primitive `float`, then null is not accepted and we simplify things a bit.